concent_api/utils/decorators.py

```diff
@@ -41,15 +45,22 @@ def require_golem_auth_message(view):
     @wraps(view)
     def wrapper(request, *args, **kwargs):
         if request.content_type == '':
+            log_string_message(logger, 'error: Content-Type is missing')
             return JsonResponse({'error': 'Content-Type is missing.'}, status = 400)
         elif request.content_type == 'application/octet-stream':
             try:
                 auth_message = load_without_public_key(request.body)
                 if isinstance(auth_message, message.concents.ClientAuthorization):
                     validate_golem_message_signed_with_key(auth_message, auth_message.client_public_key)
+                    log_message_received_in_endpoint(logger,
+                                                     request.resolver_match.view_name if request.resolver_match is not None else None,
+                                                     auth_message.__class__.__name__,
+                                                     auth_message.client_public_key)
                 else:
+                    log_string_message(logger, 'error: Client Authentication message not included')
                     return JsonResponse({'error': 'Client Authentication message not included'}, status = 400)
             except Http400 as exception:
+                log_string_message(logger, f"'error: '{exception.error_message} 'error_code: '{exception.error_code.value}")
```

Comment:
In all cases I think we should put `error_code` before `error_message`, they are usually shorter and might get lost at the end of whole string.

Reply:
done in all 2 places